src/views/start/index.jsx

```diff
@@ -54,6 +54,8 @@ export default class Start extends Component {
       ele.classList.add('loadingscreen-active');
       ele.classList.remove('loadingscreen-passive');
 
+      this.state.fileStats = {};
```

Comment:
Could we use `setState`?

```suggestion
      this.setState({fileStats:{}}
```

Reply:
This works just as well and is more corresponding to react documentation.